terratorch/models/wxc_model_factory.py

```diff
@@ -43,4 +48,5 @@ def build_model(
         **kwargs,
     ) -> Model:
         module = get_finetune_model(kwargs['model_config'])
+        print("Using wrapper.")
```

Comment:
@Joao-L-S-Almeida can we use logging here?

Reply:
Yes, see commit [b898ac3](https://github.com/IBM/terratorch/pull/179/commits/b898ac368add336fa08f5d93b9d7c944e671521c)